integrations/cache/cache-compat.php

```diff
@@ -67,7 +67,10 @@ public function add_cookie_script() {
 			'; SameSite=' . $samesite,
 			esc_js( $expiration )
 		);
-		echo "<script type='text/javascript'>\n" . $js . "</script>\n"; // phpcs:ignore WordPress.Security.EscapeOutput
+
+		$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : ' type="text/javascript"';
+
+		echo "<script{$type_attr}>\n{$js}</script>\n"; // phpcs:ignore WordPress.Security.EscapeOutput
```

Comment:
Same as in Polylang Pro https://github.com/polylang/polylang-pro/pull/1779#discussion_r1301437991
Couldn't we use printf instead?

Reply:
it seems that it misses a `\n` character just behind `{$js}` as in `PLL_Frontend_Filters_Search`
 https://github.com/polylang/polylang/blob/6ecea13398ad11cbd7dbbce5a01cfd04fbd781ec/frontend/frontend-filters-search.php#L175